docs/getting-started/github-apps/migrating-tools-to-github-apps.rst

```diff
@@ -0,0 +1,158 @@
+because of these fundamental differences between existing tools and GitHub App tools. The migration process for tools will not convert your existing tool into a GitHub App tool.
+Instead, you will be creating a new GitHub App tool, then deleting your existing tool. 
+
+GitHub App Installation
+-----------------------
+
+The first step to migrating a tool is the same as adding a new tool via GitHub apps: install our Dockstore GitHub app onto your repository or
+organization. You do this by navigating to ``/my-tools``, clicking the ``+`` button on the left hand sidebar, selecting ``Register using GitHub Apps``, and then clicking
+``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can select which repositories can be accessed by the GitHub app.
+
+.. image:: /assets/images/docs/add-tool-button.png
+   :width: 40 %
+
+.. image:: /assets/images/docs/register-tool-github-apps.png
+   :width: 40 %
+
+.. image:: /assets/images/docs/manage-gh-app-installation.png
+   :width: 40 %
+
+Creating a .dockstore.yml File
+-------------------------------
+
+Once the GitHub app is installed on the correct repo, the next step is to create a ``/.dockstore.yml`` file. We'll cover a very straightforward example
+first, but depending on how you configured the tool during registration and whether your GitHub repository houses multiple tools published on Dockstore,
+there will be additional steps to writing your ``/.dockstore.yml`` file.
+
+Let's say we have the following CWL tool registered on Dockstore that came from this `repository <https://github.com/kathy-t/dockstore-tool-helloworld>`__ and you would like to convert the master branch.
+
+.. figure:: /assets/images/docs/single-tool-to-migrate.png
+   :alt: Tool to Migrate
+
+As noted in our other documentation, create a ``/.dockstore.yml`` file in the root directory of the branch you want to migrate (in this example, it's the master branch) in your repository. The file should look like the following
+
+.. code:: yaml
+
+   version: 1.2
+   tools:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test.json
+
+The information above was filled out using the following:
+
+- ``subclass`` is based on the descriptor language your tool is described in. In this case, the tool is described in CWL.
+- ``primaryDescriptorPath`` is from ``CWL Path`` or ``WDL Path``, depending on the ``subclass``. 
+- ``testParameterFiles`` is from ``CWL Test Parameter File Path`` or ``WDL Test Parameter File Path``, depending on the ``subclass``.
+
+.. note:: 
+    Existing Dockstore tools can be described in two languages, CWL and WDL, but GitHub App tools can only be described in one language. If your existing tool is described in two languages, you can either pick one language
+    or register two GitHub App tools, one for each language.
```

Comment:
Maybe note that they could register both in one `.dockstore.yml` and/or provide an example like 
```
version: 1.2
tools:
      - subclass: CWL
        primaryDescriptorPath: /Dockstore.cwl
        testParameterFiles:
            - /test.json
      - subclass: WDL
        primaryDescriptorPath: /Dockstore.cwl
        testParameterFiles:
            - /test.json 
        name: wdl
```